utils/logutils.go

```diff
@@ -0,0 +1,12 @@
+package utils
+
+import (
+	"os"
+
+	log "github.com/Sirupsen/logrus"
+)
+
+func LogExit(msg string, fmts ...interface{}) {
+	log.Fatalf(msg, fmts...)
```

Comment:
@erikh, I am still wondering if we need this utility function. Unless user set's log level to be `PanicLevel`, logrus will exit.

also just fyi I created a PR https://github.com/Sirupsen/logrus/pull/188 in logrus to fix the above behavior, once it is merged we can update our godeps to get the fix.